backend/insert.py

```diff
@@ -14,27 +14,27 @@
 Logins_to_add = [{
     'id': 1,
     'email': 'arshallat@uni.minerva.edu',
-    'password_hash': '',
+    'password': '',
```

Comment:
In later iterations the `insert.py` would be handled by the tests?

Reply:
Currently I don't have the testing systems yet but this is a great suggestion. Will take note of this for my testing in the future